test_runner/batch_others/test_import.py

```diff
@@ -0,0 +1,183 @@
+import pytest
+from fixtures.neon_fixtures import NeonEnvBuilder
+from uuid import UUID, uuid4
+import tarfile
+import os
+import shutil
+from pathlib import Path
+import json
+from fixtures.utils import subprocess_capture
+from fixtures.log_helper import log
+from contextlib import closing
+from fixtures.neon_fixtures import pg_distrib_dir
+
+
+@pytest.mark.timeout(600)
+def test_import_from_vanilla(test_output_dir, pg_bin, vanilla_pg, neon_env_builder):
+    # Put data in vanilla pg
+    vanilla_pg.start()
+    vanilla_pg.safe_psql("create user cloud_admin with password 'postgres' superuser")
+    vanilla_pg.safe_psql('''create table t as select 'long string to consume some space' || g
+     from generate_series(1,30000000) g''')
+    assert vanilla_pg.safe_psql('select count(*) from t') == [(30000000, )]
+    # ensure that relation is larger than 1GB to test multisegment restore
+    assert vanilla_pg.safe_psql("select pg_relation_size('t')")[0][0] > 1024 * 1024 * 1024
+
+    # Take basebackup
+    basebackup_dir = os.path.join(test_output_dir, "basebackup")
+    base_tar = os.path.join(basebackup_dir, "base.tar")
+    wal_tar = os.path.join(basebackup_dir, "pg_wal.tar")
+    os.mkdir(basebackup_dir)
+    vanilla_pg.safe_psql("CHECKPOINT")
+    pg_bin.run([
+        "pg_basebackup",
+        "-F",
+        "tar",
+        "-d",
+        vanilla_pg.connstr(),
+        "-D",
+        basebackup_dir,
+    ])
+
+    # Make corrupt base tar with missing pg_control
+    unpacked_base = os.path.join(basebackup_dir, "unpacked-base")
```

Comment:
I see strange artifact in CI run:

[tmp/test_output/test_import_from_vanilla/basebackup/unpacked-base/pg.log](https://output.circle-artifacts.com/output/job/e701e0fe-03c8-4cb5-b5b8-aca404142838/artifacts/0/tmp/test_output/test_import_from_vanilla/basebackup/unpacked-base/pg.log)

What process starts postgres in this directory? I thought it was a temp dir for a tar...

Reply:
`pg.log` is unpacked from `base.tar`